build.sbt

```
@@ -30,9 +30,12 @@ lazy val core = project.settings(
     "org.scalatest" %% "scalatest" % "3.2.16" % Test,
     "org.scalatestplus" %% "scalacheck-1-17" % "3.2.16.0" % Test
   ),
-  Test/testOptions += Tests.Argument(
-    TestFrameworks.ScalaTest,
-    "-u", s"test-results/scala-${scalaVersion.value}"
+  Test / testOptions ++= Seq(
+    Tests.Argument(TestFrameworks.ScalaTest, "-o"),
+    Tests.Argument(
+      TestFrameworks.ScalaTest,
+      "-u", s"test-results/scala-${scalaVersion.value}"
+    )
```

`Tests.Argument` actually accepts multiple arguments, so we can reduce the config a little bit by putting the config in one entry like [this](https://github.com/guardian/google-search-indexing-observatory/blob/587cdf235f5536993962266a53ad7175d33cee3d/build.sbt#L46):

```suggestion
  Test / testOptions +=
    Tests.Argument(TestFrameworks.ScalaTest, "-u", s"test-results/scala-${scalaVersion.value}", "-o")
```

---

Ah lovely!